HdcpProfile/HdcpProfile.cpp

```diff
@@ -55,6 +59,10 @@ namespace WPEFramework
 
             registerMethod(HDCP_PROFILE_METHOD_GET_HDCP_STATUS, &HdcpProfile::getHDCPStatusWrapper, this);
             registerMethod(HDCP_PROFILE_METHOD_GET_SETTOP_HDCP_SUPPORT, &HdcpProfile::getSettopHDCPSupportWrapper, this);
+
+            Register("setApiVersionNumber", &HdcpProfile::setApiVersionNumberWrapper, this);
```

Comment:
Pls remove these APIs. You can refer to https://github.com/rdkcentral/rdkservices/commit/3692632373e8e82dba92bec56f9e6082b430a829#diff-f6cd28bb8911a0253a4601f823d3777089aecd54eb214bc6cdc227961da7b13f on how versioning is implemented in other plugins.